presto-orc/src/main/java/com/facebook/presto/orc/OrcSelectiveRecordReader.java

```diff
@@ -729,6 +735,21 @@ else if (!hasAnyFilter(columnIndex)) {
         return page;
     }
 
+    private long getSelfRetainedSizeInBytes()
+    {
+        return INSTANCE_SIZE +
```

Comment:
this memory is quite negligible, do you think we will need to report memory from these? I don't see the same for OrcBatchRecordReader

Reply:
Lets keep this. This is consistently reported across all stream readers. I would think it is better to overreport than to underreport and cause instability.

Reply:
@bhhari `OrcBatchRecordReader` doesn't hold these arrays. For the `OrcSelectiveRecordReader` these fields could be dozens of KB. Considering number of drivers and query parallelism, this can be MBs. It's not big memory but it's better to count them.